app/util.go

```diff
@@ -84,11 +86,13 @@ func SortJSON(toSortJSON []byte) string {
 	var c interface{}
 	err := json.Unmarshal(toSortJSON, &c)
 	if err != nil {
-		panic(err)
+		fmt.Println("could not unmarshal json in SortJSON: " + err.Error())
+		os.Exit(1)
 	}
 	js, err := json.MarshalIndent(c, "", "    ")
 	if err != nil {
-		panic(err)
+		fmt.Println("could not marshal back to json in SortJSON: " + err.Error())
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("could not marshal back to json in SortJSON: %s",  err.Error())
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)